spec/models/manageiq/providers/amazon/aws_refresher_spec_common.rb

```diff
@@ -1101,4 +1103,9 @@ def assert_specific_vm_in_other_region
   def assert_relationship_tree
     expect(@ems.descendants_arranged).to match_relationship_tree({})
   end
+
+  def assert_specific_tags_on_vm
+    vm = ManageIQ::Providers::Amazon::CloudManager::Vm.where(:name => "EmsRefreshSpec-PoweredOn-Basic3").first
+    expect(vm.tags).to be_empty
+  end
```

Comment:
This isn't really testing anything, can you pick a VM with tags? Given the feature freeze, I am fine with merging as is if a more robust test is implemented in a follow up PR.